src/maxKey.js

```diff
@@ -9,14 +9,26 @@
  * @returns {number} The key with the largest value
  */
 function maxKey(obj) {
-  // This is your job. :)
+  let values = Object.values(obj);
+  let keys = Object.keys(obj);
+  let largestValue = values[0]
+  let largestValueKey = keys[0]
+
+  for (let key of keys){
+    if (obj[key] > largestValue) {
+      console.log('the value:', obj[key])
```

Comment:
Remove debugging `console.log` statements before submitting things for feedback.